.github/workflows/cucumber-tests.yml

```diff
@@ -0,0 +1,20 @@
+name: Cucumber Tests
+
+on:
+  push:
+    branches: [master]
```

Comment:
question: so this only runs on push to master? so we wont know if tests are failing until after the merge of a PR right? dont we want to check the status of the tests prior to code going to master?

Reply:
Test are currently too slow to run on pull requests, it takes about 15mn to run and it's due to docker-compose up, once we figure it out we could move it down to pr.